data_structures/src/mainnet_validations.rs

```diff
@@ -209,6 +209,34 @@ pub fn after_second_hard_fork(epoch: Epoch, environment: Environment) -> bool {
     epoch >= SECOND_HARD_FORK && Environment::Mainnet == environment
 }
 
+/// Allows to check the active Witnet Improvement Proposals
+#[derive(Clone, Debug)]
+pub struct ActiveWips {
```

Comment:
Is this struct similar to TapiEngine?

Reply:
The `active_wips` field is exactly the same as `tapi_engine.wip_activation`, but the other fields are not the same.

Reply:
should we include the other fields and the impl in the TapiEngine?